backend/src/routes/login.route.ts

```diff
@@ -15,8 +16,12 @@ export class LoginRoute {
 
   private initializeRoutes() {
     this.router.post(this.path, async (req, res) => {
+
+      const requestValidity = this.validateRequest(req.body);
+      if (requestValidity.error) return res.status(404).send(requestValidity.error.message);
+        
       const user = await this.users.findOne({
-        email: req.body.email,
+        email: req.body.email.toLowerCase(),
```

Comment:
Do you want to put this into the JOI validation?

Reply:
Got it working there